mlflow/h2o.py

```diff
@@ -1,4 +1,12 @@
-"""MLflow integration for H2O."""
+"""
+The ``mlflow.h2o`` module provides an API for logging and loading H2O models. This module supports
+exporting H2O models with the following flavors:
+
+H20 (native) format
+    This is the main flavor and is always produced.
```

Comment:
We actually always produce both flavors. Downstream, one or the other will be picked -- for example, the pyfunc version would be used by SageMaker for deployment (as SageMaker knows how to deploy pyfuncs, but not H2O models), while the H2O format could be loaded by the user to use H2O-specific functionality.